### ISSUE TITLE
[HELP] UNCHARTED failed to parse patchlist

### ISSUE THREAD

**Issue by sumimasen03** [Electry/VitaGrafixPatchlist#91]
I've already copy the patchlist then everytime i open uncharted. vitagrafix error failed to open patch file.
 I already installed ioPlus.

**Comment by Keivin1009** [comment]
I have JP Version [PCSC00005], but no patchlist here....

**Comment by Electry** [comment]
are you on 5.0.2? If not then update, if yes send me ux0:data/VitaGrafix/log.txt

**Comment by sumimasen03** [comment]
![2020-03-23-155955-546879](https://user-images.githubusercontent.com/61772446/77295063-3abbf900-6d20-11ea-9bdb-c4db4d2b6cbd.jpg)

**Comment by sumimasen03** [comment]
![2020-03-23-160103-346188](https://user-images.githubusercontent.com/61772446/77295113-4c050580-6d20-11ea-973a-d3daecd9db3b.jpg)

**Comment by TGB-KillSpam** [comment]
After a few hours of searching everywhere about this error i finally found a fix by myself.
ux0:data/VitaGrafix/patch/PCSF0001.txt it's where we need to focus, there's where the patchfile need to be, not in the patchfiles, we need to make a need folder in   ux0:data/VitaGrafix/   called "patch"  with the patch info in it : 
     https://raw.githubusercontent.com/Electry/VitaGrafixPatchlist/master/patch/PCSF00001.txt 
![step 1](https://user-images.githubusercontent.com/68359106/87615515-d37ce700-c70a-11ea-9a8a-78633f68c2c4.png)
![step 2](https://user-images.githubusercontent.com/68359106/87615524-d7106e00-c70a-11ea-9273-3ff505922eca.png)

Basically the error code it's telling us what to do, this is how i figured it out, i thought it's worth a try so i ended up fixing it. 
![77295063-3abbf900-6d20-11ea-9bdb-c4db4d2b6cbd](https://user-images.githubusercontent.com/68359106/87615714-42f2d680-c70b-11ea-8c44-202800662c2c.jpg)

Hope i helped you guys, sorry if couldn't  explain it very good, i don't have much experience on forums but i just wanted to help.
Let me know if this helped you. 
Thanks!
